### PR TITLE
add driver,homepage, exe

### PR DESCRIPTION
**Pull request by AlesiaMianitskaya**

### CODE REVIEW

File: page/HomePage.java [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+  @FindBy(xpath = "//div/a[@data-name='Infants']")
+  private WebElement buttonInfants;
+
+  @FindBy(xpath = "//div[@class='half infants']//a[@data-value='2']")
+  private WebElement selectInfants;
+
+  @FindBy(xpath = "//div[@id='flights-container']//a[@data-method='Flights']")
+  private WebElement buttonFindFlights;
+
+  @FindBy(xpath = "//div[@class='modal fade search-panel-modal validation-errors in']//p[@style='display: block;']")
```
Comment (DzmitryZakharenka) [inline page/HomePage.java]: Must we use xpath with so many classes? Do we sure that this div always has all this classes? 
It's okay, but in the future we must avoid so many contained classes into xpath.

File: page/HomePage.java [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    super(driver);
+    PageFactory.initElements(this.driver, this);
+  }
+
+  @Override
+  public void open() {
+    driver.get(HOME_URL);
+  }
+
+  public void selectFrom() {
```
Comment (DzmitryZakharenka) [inline page/HomePage.java]: All pageObject classes must contain only logic scenarios as public methods - like login or register or findTravel(some parameters) in our case. We think we must create bean like TravelParameters class in the future.

File: page/HomePage.java [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+  public void selectChildren(){
+    buttonChildren.click();
+    selectChildren.click();
+  }
+
+  public void findFlights(){
+    buttonFindFlights.click();
+  }
+
+  public boolean isAlertMessage(){
```
Comment (DzmitryZakharenka) [inline page/HomePage.java]: Maybe isAlertDisplayed will be better name for this case?

File: driver/DriverSingleton.java [hunk 4]
```diff
@@ -0,0 +1,30 @@
+package com.epam.framework.driver;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+import java.util.concurrent.TimeUnit;
+
+public class DriverSingleton {
+  private static WebDriver driver;
```
Comment (DzmitryZakharenka) [inline driver/DriverSingleton.java]: We understand that it is our fault that we doesn't discuss this before, but let's follow Google code style convention. In this case - we must use 4 whitespaces for indents.

File: page/HomePage.java [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+
+
+public class HomePage extends AbstractPage {
+
+  private final String HOME_URL = "http://www.monarch.co.uk/";
+
+  @FindBy(xpath = "//div/a[@data-name='OriginAirport']")
+  private WebElement buttonFrom;
+
+  @FindBy(linkText = "Birmingham") //xpath = "//div//a[@data-value='BHX']"
```
Comment (AlesiaMianitskaya) [inline page/HomePage.java]: done

File: page/HomePage.java (hunk 3 above)
Comment (AlesiaMianitskaya) [inline page/HomePage.java]: done

File: page/HomePage.java (hunk 5 above)
Comment (AlesiaMianitskaya) [inline page/HomePage.java]: done

File: page/HomePage.java [hunk 6]
```diff
@@ -0,0 +1,87 @@
+package com.epam.framework.page;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
```
Comment (AlesiaMianitskaya) [inline page/HomePage.java]: done

File: driver/DriverSingleton.java [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+
+  private DriverSingleton() {
+  }
+
+  public static WebDriver getDriver() {
+    if (driver == null) {
+      System.setProperty(DRIVER, PATH_DRIVER );
+      driver = new ChromeDriver();
+      driver.manage().timeouts().pageLoadTimeout(10, TimeUnit.SECONDS);
+      driver.manage().timeouts().implicitlyWait(10, TimeUnit.SECONDS);
```
Comment (AlesiaMianitskaya) [inline driver/DriverSingleton.java]: done

File: driver/DriverSingleton.java (hunk 4 above)
Comment (AlesiaMianitskaya) [inline driver/DriverSingleton.java]: done

File: driver/DriverSingleton.java [hunk 8]
```diff
@@ -0,0 +1,30 @@
+package com.epam.framework.driver;
```
Comment (AlesiaMianitskaya) [inline driver/DriverSingleton.java]: done

File: page/HomePage.java (hunk 2 above)
Comment (AlesiaMianitskaya) [inline page/HomePage.java]: Done

**Pull request merged**